--- a/data/appdata/org.gnome.ColorViewer.appdata.xml.in	2019-02-08 12:51:55.000000000 +0100
+++ a/data/appdata/org.gnome.ColorViewer.appdata.xml.in	2020-01-15 17:19:39.832180015 +0100
@@ -19,10 +19,10 @@
   </description>
   <screenshots>
     <screenshot type="default">
-      <image>https://git.gnome.org/browse/gnome-color-viewer/plain/data/appdata/ss-info.png</image>
+      <image>https://gitlab.gnome.org/hughsie/gnome-color-viewer/raw/master/data/appdata/ss-info.png</image>
     </screenshot>
     <screenshot>
-      <image>https://git.gnome.org/browse/gnome-color-viewer/plain/data/appdata/ss-cie1931.png</image>
+      <image>https://gitlab.gnome.org/hughsie/gnome-color-viewer/raw/master/data/appdata/ss-cie1931.png</image>
     </screenshot>
   </screenshots>
   <releases>
@@ -35,6 +35,7 @@
       </description>
     </release>
   </releases>
+  <content_rating type="oars-1.0" />
   <url type="homepage">https://gitlab.gnome.org/hughsie/gnome-color-viewer</url>
   <url type="bugtracker">https://gitlab.gnome.org/hughsie/gnome-color-viewer/issues</url>
   <url type="donation">http://www.gnome.org/friends/</url>
